Datagrid/ProxyQuery.php

```diff
@@ -74,6 +74,24 @@ public function execute(array $params = array(), $hydrationMode = null)
             $queryBuilder->resetDQLPart('orderBy');
         }
 
+        /* By default, always add a sort on the identifier fields of the first
+            * used entity in the query, because RDBMS do not guarantee a
```

Comment:
Doesn't it? Vim does this when using `gqq` I was wondering if it was a standard or something, but since it looks weird to you too, I guess it probably is not. Anyway, I won't  be able to changr that soon because of holidays, sorry.